.github/workflows/sycl-linux-precommit.yml

```diff
@@ -47,9 +47,28 @@ jobs:
       build_image: "ghcr.io/intel/llvm/ubuntu2204_build:7ed894ab0acc8ff09262113fdb08940d22654a30"
       changes: ${{ needs.detect_changes.outputs.filters }}
 
-  test:
+  determine_arc_tests:
+    name: Decide which Arc tests to run
     needs: [build, detect_changes]
     if: ${{ always() && !cancelled() && needs.build.outputs.build_conclusion == 'success' }}
+    # Github's ubuntu-* runners are slow to allocate. Use our CUDA runner since
+    # we don't use it for anything right now.
```

Comment:
We use it for CUDA in nightly, and might use it for CTS runs as well :)

Reply:
I just copied this comment from another file. Should I use a different runner? I have no idea what the infra allocation looks like.

Reply:
Just drop it. I think we mostly use self-hosted runners anyway, so it's not an exception here.

Reply:
Can be a separate PR.

Reply:
> Can be a separate PR because the testing is green. Or push the update and merge without waiting for the results ;)

Reply:
Will do separate PR